images/router/haproxy/conf/haproxy-config.template

```diff
@@ -34,7 +34,7 @@
 
 {{- /* hsts header in response: */}}
 {{- $hstsOptionalTokenPattern := `(?:includeSubDomains|preload)` }}
-{{- $hstsPattern := printf `(?:%[1]s[;])*max-age=(?:\d+|"\d+")(?:[;]%[1]s)*`  $hstsOptionalTokenPattern -}}
+{{- $hstsPattern := printf `(?i)(?:%[1]s\s*[;]\s*)*max-age=(?:\d+|"\d+")(?:\s*[;]\s*%[1]s)*`  $hstsOptionalTokenPattern -}}
```

Comment:
Nice.  Does the spec allow spaces around the equal sign?

Reply:
Good question!
Looks like RFC2616 is allowing LWS (linear white spaces) between any word and separator (`=` is also considered as separator): [here](https://datatracker.ietf.org/doc/html/rfc2616#section-2.1) (`implied *LWS` notion).
Didn't see examples with LWS around `=` though.